pkg/installer/check.go

```diff
@@ -201,14 +201,14 @@ func (o *Installer) ProviderURLParse(path string, acceptPreRelease bool) (packag
 					}
 
 					if packageURL == "" {
-						packageURL = fmt.Sprintf("https://github.com/%s/%s/releases/download/%s/%s-%s-%s.tar.gz",
-							o.Org, o.Repo, version, o.Name, o.OS, o.Arch)
+						packageURL = fmt.Sprintf("https://github.com/%s/%s/releases/download/%s/%s-%s-%s.%s",
+							o.Org, o.Repo, version, o.Name, o.OS, o.Arch, getPackagingFormat(o))
```

Comment:
About resuing the existing variable which defined in line 154

Reply:
Thanks for the catch. Made the required changes